Samsung/A51/res/values/config.xml

```diff
@@ -1,633 +1,98 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-	    <string-array name="config_tether_usb_regexs">
-        <item>rndis0</item>
+    <integer name="config_screenBrightnessSettingMaximum">255</integer>
```

Comment:
This value was already there...? Why move it away? It makes re-reading pretty hard